CHANGELOG.md

```diff
@@ -5,6 +5,13 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+## [8.6.0]
+### Added
+- feat: add deeplinks for Send Eth, Send ERC20 and Approve ERC20 ([#313](https://github.com/MetaMask/test-dapp/pull/313))
+- update NFT contract ([#315](https://github.com/MetaMask/test-dapp/pull/315))
+- feat: add get allowance support for ERC20 tokens ([#312](https://github.com/MetaMask/test-dapp/pull/312))
+- fix: Fix network status field ([#306](https://github.com/MetaMask/test-dapp/pull/306))
```

Comment:
maybe I would move the below 2 entries to a new section for the fixed items:

### Fixed
- Fix network status field ([#306](https://github.com/MetaMask/test-dapp/pull/306))
- Fix NFT contract address for Base network ([#315](https://github.com/MetaMask/test-dapp/pull/315))